src/components/App/app.less

```diff
@@ -12,7 +8,7 @@ td {
 }
```

Comment:
You can use > selector for a bit better performance (which is almost not noticable, but still worth...)